docs/2.7.0/versions.json

```diff
@@ -1,5 +1,5 @@
 {
   "daml": "2.7.0-snapshot.20230523.11810.0.06abd5c6",
   "canton": "2.7.0-snapshot.20230531.10456.0.v26f31ff7",
-  "daml_finance": "1.2.14"
+  "daml_finance": "1.2.14.99.20230615.1"
```

Comment:
It would be nice if you could align your snapshot numbers on what seems to have become the company standard.

Reply:
You mean something similar to "2.7.0-snapshot.20230531.10456.0.v26f31ff7"?
Or is the last part sufficient? "-snapshot.20230531.10456.0.v26f31ff7"?

Reply:
I meant the snapshot format (i.e. the latter), but aligning on minor version would also be good.

Reply:
(Obviously I'm not expecting either to be done as part of this PR.)

Reply:
Thanks, I will discuss it with the team.

Reply:
Fyi, I have opened an issue for this:
https://github.com/digital-asset/daml-finance/issues/879
Seems that version number is only used in that place, and it is not configurable without some development.